### ISSUE TITLE
1.1.2 - Cast Error on getBondedDevices

### ISSUE THREAD

**Issue by Cadey** [kakzaki/blue_thermal_printer#69]
Hi,

I have just updated to 1.1.2 and the following error happens when I call getBondedDevices()

```
_CastError (type 'Future<dynamic>' is not a subtype of type 'FutureOr<List<dynamic>>' in type cast)

final List list = await (_channel.invokeMethod('getBondedDevices') as FutureOr<List<dynamic>>);
```

When I changed the code as follows, it works as expected
```
final List list = (await _channel.invokeMethod('getBondedDevices') as List<dynamic>);
```

**Comment by excode** [comment]
I a getting the same error and failed to  use printer. any help or update available for this issue??

**Comment by excode** [comment]
FYI . I am using null-safety version

**Comment by kakzaki** [comment]
thanks for the information, i fixed it at ver 1.1.3
